### PR TITLE
First rendering of Registration Page

### PR DESCRIPTION
**Pull request by jlrusso**
image

### CODE REVIEW

File: resources/js/Pages/Register.vue [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+        if (this.hasEmptyRequiredFields()) {
+          throw new Error('Please fill out all required fields.');
+        }
+
+        if (this.hasFieldsUnderFiveChars()) {
+          throw new Error('Please use field values greater than 5 characters.');
+        }
+
+        if (this.hasInvalidEmailFormat()) {
+          throw new Error('Please fill all required fields with at least 5 characters and valid email.');
```
Comment (jlrusso) [inline resources/js/Pages/Register.vue]: will reduce this to only mention email validity

File: app/Http/Controllers/RegisterController.php [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+class RegisterController extends Controller
+{
+    public function showRegister(Request $request): InertiaResponse
+    {
+        return Inertia::render('Register');
+    }
+
+    public function register(Request $request): InertiaResponse
+    {
+        dd($request->all());
```
Comment (joshuaGlass808) [inline app/Http/Controllers/RegisterController.php]: what's going on here?

File: resources/js/Pages/Register.vue (hunk 1 above)
Comment (joshuaGlass808) [reply resources/js/Pages/Register.vue]: this is fine.

File: resources/js/Pages/Register.vue [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+      </section>
+    </div>
+  </Layout>
+</template>
+
+<style lang='scss' scoped>
+  #registration-page {
+    width: 100vw;
+    min-height: calc(100vh - 60px);
+    background: #d5d8dc;
```
Comment (joshuaGlass808) [inline resources/js/Pages/Register.vue:186]: not quite sold on #d5d8dc, but its fine for now.

**Review by joshuaGlass808 — APPROVED** [review]

**Pull request merged**